### DISCUSSION TITLE
Use a better colour palette for callouts

### DISCUSSION THREAD

**Discussion by routerino** [outline/outline#1978] [Ideas]
A problem with the warning and tip callouts is that they are extremely eye jarring. I avoid using them almost entirely because they are too jarring and eye catching compared to the rest of the content. Their contrast is also extremely high which makes actually reading the text difficult.

I would suggest an approach that bookstack takes, with more subtle backgrounds and less contrast between the background and text, like so:
![image](https://user-images.githubusercontent.com/45954722/111953366-9c34c480-8b3a-11eb-9bb0-59034fa61c1d.png)

**Comment by tommoor** [comment]
Yea, I can see that logic – we maybe went a little _too_ saturated on the alert palette 😄
